src/tools/kdb/external.cpp

```diff
@@ -52,11 +52,17 @@ void tryExternalCommand (char ** argv)
 {
 	std::vector<std::string> pathes;
 
-	char * execPath = getenv ("KDB_EXEC_PATH");
-	if (execPath)
+	std::string execPath = getenv ("KDB_EXEC_PATH");
+	size_t pos = 0;
+	std::string token;
+	std::string delimiter = ":";
+	while ((pos = execPath.find (delimiter)) != std::string::npos)
 	{
-		pathes.push_back (execPath);
+		token = execPath.substr (0, pos);
+		pathes.push_back (token);
+		execPath.erase (0, pos + delimiter.length ());
 	}
+	pathes.push_back (execPath);
```

Comment:
Thank you for implementing this!

What happens with no KDB_EXEC_PATH given?

Reply:
> What happens with no KDB_EXEC_PATH given?

I assume this lead to some errors and hope it is fixed in the next commit.